src/routes/opendata.js

```diff
@@ -0,0 +1,132 @@
+const OpenDataFacultyRepository = require("../repository/conUOpenDataFacultyRepository");
+const OpenDataCourseRepository = require("../repository/conUOpenDataCourseRepository");
+const OpenDataImportantDateRepository = require("../repository/conUOpenDataImportantDateRepository");
+const router = require('express').Router();
+
+
+/** runs once on server start then refreshes open data once every 24 hours (if --odrefreshinterval param is not specified;
+ * else refreshes every 'odrefreshinterval' ms); open data refresh is not run if --odrefresh param is not set to true
+ */
+if (process.env.npm_config_odrefresh === "true")
+    setInterval(
+        function openDataRefresh() {
+            OpenDataFacultyRepository.refreshFacultyData();
+            OpenDataCourseRepository.refreshCourseData();
+            OpenDataImportantDateRepository.refreshImportantDateData();
+
+            return openDataRefresh;
+        }(), (process.env.npm_config_odrefreshinterval && Number.isInteger(Number(process.env.npm_config_odrefreshinterval)) ? Number(process.env.npm_config_odrefreshinterval) : 86400000));
+
+/**
+ * add a faculty to a department
+ * @param {facultyCode, facultyDescription, departmentCode, departmentDescription}: passed as request body and added to the opendatafaculties collection
```

Comment:
what you put inside the curly bracket is the type not the var name. put all these on different line or use data.facultyCode